addons/l10n_pk/demo/account_demo.py

```diff
@@ -0,0 +1,324 @@
+                            ])],
+                        }),
+                    ]
+                },
+                'demo_vendor_bill_2': {
+                    'ref': 'INV/002',
+                    'move_type': 'in_invoice',
+                    'partner_id': 'res_partner_islamabad',
+                    'invoice_user_id': 'base.user_demo',
+                    'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                    'invoice_date': fields.Datetime.today() - relativedelta(days=2),
+                    'invoice_line_ids': [
+                        Command.create({
+                            'product_id': 'product.product_product_8',
+                            'quantity': 4,
+                            'price_unit': 1000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_17')
+                            ])],
+                        }),
+                        Command.create({
+                            'product_id': 'product.product_product_7',
+                            'quantity': 3,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10')
+                            ])],
+                        }),
+                    ]
+                },
+                'demo_vendor_bill_3': {
+                    'ref': 'INV/003',
+                    'move_type': 'in_invoice',
+                    'partner_id': 'res_partner_islamabad',
+                    'invoice_user_id': 'base.user_demo',
+                    'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                    'invoice_date': fields.Datetime.today() - relativedelta(days=3),
+                    'invoice_line_ids': [
+                        Command.create({
+                            'product_id': 'product.product_product_9',
+                            'quantity': 1,
+                            'price_unit': 1000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_17')
+                            ])],
+                        }),
+                        Command.create({
+                            'product_id': 'product.product_product_10',
+                            'quantity': 1,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10'),
+                                _get_tax_by_id('pk_sales_tax_services_1')
+                            ])],
+                        }),
+                    ]
+                },
+            }
+        else:
+            return super()._get_demo_data_move(company)
+
+    @api.model
+    def _get_demo_data_config_settings(self, company=False):
```

Comment:
This might Override the method in `l10n_in`

Reply:
I reverted the changes made by renaming the methods. Instead, I applied a conditional check on the `country_id`, which I believe is a more preferable and cleaner approach ☺️

Reply:
You won't get super method 👀 unless l10n_in is installed, that's the reason I asked for renaming the method / defining a inheritable method in `account` module which can be utilized by both localisation. 

![image](https://github.com/user-attachments/assets/a5a0cdc3-cdd5-45d5-bafa-60e369255d24)